### PR TITLE
Added CS498GC

### PR DESCRIPTION
**Pull request by kteal**
Added CS498GC, also fixed the CS course list

### CODE REVIEW

File: docs/Course Wiki/CS Course Offerings/CS498GC.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+This course is an introduction to the foundational principles of mobile robotics, especially from a software perspective. It begins with background in linear algebra, state spaces, and software development. Then it covers mobile (aerial and ground) robot dynamics with discussions about kinematics and coordinate transforms. After this, it covers GPS, sensor fusion, exteroceptive and proprioceptive sensors, and Kalman filters. Next, you will discuss computer vision, camera models, machine learning, and SLAM (Simultaneous Localization and Mapping). Time permitting, there may be more discussions on control theory, path planning, and motion planning.
+
+Much of this course is designed as a survey into modern mobile robotics, so the topics are not covered exhaustively and are instead meant to give you a taste of the field. Much of the focus of this course is on implementing the theory into code, so you will be using Python and ROS to implement the concepts discussed in lecture.
+
+## Prerequisites
+
+- [MATH221](../MATH%20Course%20Offerings/MATH221.md), MATH225 or equivalent, [MATH285](../MATH%20Course%20Offerings/MATH285.md) or equivalent, STAT400 or equivalent, [CS225](./CS225.md)
```
Comment (aidanrodgers) [inline docs/Course Wiki/CS Course Offerings/CS498GC.md]: Separate each prerequisite in separate bullet points, and link each page, even if they don't exist yet.  STAT400 will be in "other course offerings".  Also link them on first mention in the next paragraph

File: docs/Course Wiki/CS Course Offerings/CS498GC.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+CS498GC focuses on the software development of mobile robotics, so it is important to have a solid understanding of data structures and algorithms. Although the official prerequisite is only CS225, the syllabus states that "students are expected to have taken introductory differential equations, introduction to probability and statistics, linear algebra, and software programming." Generally, these preqrequisites are filled by [MATH257](../MATH%20Course%20Offerings/MATH257.md), MATH285, STAT400 or [ECE313](../ECE%20Course%20Offerings/ECE313.md), and CS225. Graduate standing or instructor approval can be used to waive these prerequisites.
```
Comment (aidanrodgers) [inline docs/Course Wiki/CS Course Offerings/CS498GC.md]: prerequisite misspelled

File: docs/Course Wiki/CS Course Offerings/CS498GC.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Life After
+
+This course covers a lot of topics quite briefly, so which courses you decide to take after depend on which areas you're interested in. Some courses that are related include [ECE486](../ECE%20Course%20Offerings/ECE486.md), [ECE470](../ECE%20Course%20Offerings/ECE470.md), [ECE484](../ECE%20Course%20Offerings/ECE484.md), [ECE448](../ECE%20Course%20Offerings/ECE448.md), and [CS588](./CS588.md).
+
+## Resources
+
+- [Textbook](https://i-share-uiu.primo.exlibrisgroup.com/permalink/01CARLI_UIU/gpjosq/alma99955006309205899)
+- [Fall 2023 Website](http://daslab.illinois.edu/coursesfall2023.html#cs498)
+- [Fall 2023 Syllabus](https://uofi.app.box.com/s/ag4j39jnall7g5lr3xvrxb771cf6hnpb)
```
Comment (aidanrodgers) [inline docs/Course Wiki/CS Course Offerings/CS498GC.md]: don't link website/syllabus if they will change in future semesters

**Review by aidanrodgers — CHANGES_REQUESTED** [review]
small changes

**Comment by kteal** [comment]
since the cs courses have been updated since I made this branch, the list is out of date again. should I merge master into this branch and then add those classes to the list or can we tackle that in a future PR?

**Comment by aidanrodgers** [comment]
> since the cs courses have been updated since I made this branch, the list is out of date again. should I merge master into this branch and then add those classes to the list or can we tackle that in a future PR?

Don’t worry about this, I can update it.

**Review by aidanrodgers — APPROVED** [review]

**Pull request merged**